rust/ares/src/noun.rs

```diff
@@ -150,6 +150,11 @@ impl fmt::Debug for DirectAtom {
     }
 }
 
+#[allow(non_snake_case)]
+pub fn D(n: u64) -> Noun {
```

Comment:
```suggestion
pub const fn D(n: u64) -> Noun {
```

This signals to the compiler that this function can be evaluated at compile time.